docs/post/memcards.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Memory Cards"
+date: 2024-03-20
+summary: "This guides shows you how to import save data to your virtual memory card"
+draft: false
+toc: true
+sidebar_position: 4
+---
+
+This section shows you how to import save data to your virtual memory card
+
+## General
+
+Most save files for PlayStation 2 games during the console's lifecycle were extracted from consoles using the various cheat devices which were sold at the time. Some examples are Action Replay Max, Code Breaker, and GameShark. However, because these companies were competing and did not want any other companies to be able to use saves extracted with their devices, the saves were repackaged into proprietary formats and encrypted. To use these save files on PCSX2, they must be decrypted and unpackaged by specialized software.
+
+## What is MyMC?
+
+MyMC is software designed to interact with PlayStation 2 memory card images (.ps2) as was used by PCSX2. It allows you to add, remove, as well as export save data.
+
+Multiple versions of MyMC have been created over the years. The first release of MyMC was by Ross Ridge, who is responsible for documenting a substantial amount of information on PS2 memory cards. Subsequent updates have been made by other authors, porting the program to a native Python app, and adding support for newer container formats.
+
+### MyMC by Ross Ridge
+
+Ross Ridge's original MyMC program works without issue on Windows 10 and newer and is a Windows-exclusive program. However, it predates the wide availability of PSV files – the format used to extract PS2 saves from a PS3 – and does not have support for these. For non-advanced users on Windows, this is probably the most straightforward version to run, as it is a basic Windows `.exe` file.
+
+### MyMC+ Python port by thestr4ng3r
+
+A Python 3 native port was created by thestr4ng3r which made some adjustments and added some additional features, most notably PSV support. While this version is cross-platform and more feature-complete, we generally feel it has been obsoleted due to some compatibility issues it had with Python libraries such as wxPython, and it has been improved upon by a newer release. We generally recommend not using this version, but it is important to remember it as a stepping stone for the latest revision.
+
+### MyMC++ Python port by Adubbz
+
+A second Python 3 native port is currently our recommended choice for users who understand how to follow more detailed setup instructions. This version brings in the improvements from MyMC+ such as cross-platform support and PSV support, adds additional usability improvements, and reliably runs on the latest versions of Python 3. This version also adds support for adding save files to the .mc2 memory card format used by the MemCard PRO2.
```

Comment:
"for users who understand how to follow more detailed setup instructions"

This was originally my line so I take responsibility for this, but it is condescending on review. Probably could just replace with:

"for advanced users"